dfx.json

```diff
@@ -24,6 +24,7 @@
       "wasm": "target/wasm32-unknown-unknown/release/wallet-opt.wasm"
     }
   },
+  "dfx": "0.8.4",
```

Comment:
I'd rather avoid this so we don't need to keep updating the version repeatedly
```suggestion
```

Reply:
@krpeacock for documentation about XDR declaration, is it best to have that in the README or CHANGELOG file? or both?